controllers/aliases.js

```diff
@@ -63,28 +63,28 @@ const aliases = {
     '/funding/funding-guidance/applying-for-funding/full-cost-recovery': '/funding/funding-guidance/full-cost-recovery',
     '/funding/funding-guidance/applying-for-funding/help-using-our-electronic-application-forms': '/funding-guidance/help-using-our-application-forms',
     '/funding/funding-guidance/applying-for-funding/information-checks': '/funding/funding-guidance/information-checks',
-    '/funding/funding-guidance/managing-your-funding/grant-acknowledgement-and-logos/logodownloads': '/funding/funding-guidance/managing-your-funding/grant-acknowledgement-and-logos',
+    '/funding/funding-guidance/managing-your-funding/evaluation': '/funding/funding-guidance/evaluation',
     '/funding/funding-guidance/managing-your-funding/grant-acknowledgement-and-logos/LogoDownloads': '/funding/funding-guidance/managing-your-funding/grant-acknowledgement-and-logos',
+    '/funding/funding-guidance/managing-your-funding/grant-acknowledgement-and-logos/logodownloads': '/funding/funding-guidance/managing-your-funding/grant-acknowledgement-and-logos',
     '/funding/funding-guidance/managing-your-funding/help-with-publicity': '/funding/funding-guidance/managing-your-funding',
     '/funding/funding-guidance/managing-your-funding/logodownloads': '/funding/funding-guidance/managing-your-funding/grant-acknowledgement-and-logos',
+    '/funding/funding-guidance/managing-your-funding/monitoring-forms': '/funding/funding-guidance/monitoring-forms',
     '/funding/funding-guidance/managing-your-funding/ordering-free-materials/bilingual-materials-for-use-in-wales': '/funding/funding-guidance/managing-your-funding/ordering-free-materials',
     '/funding/funding-guidance/managing-your-funding/self-evaluation': '/funding/funding-guidance/evaluation',
-    '/funding/funding-guidance/managing-your-funding/evaluation': '/funding/funding-guidance/evaluation',
-    '/funding/funding-guidance/managing-your-funding/monitoring-forms': '/funding/funding-guidance/monitoring-forms',
     '/funding/past-grants': '/funding/grants',
     '/funding/past-grants/search': '/funding/grants',
-    '/funding/search-past-grants*': '/funding/grants',
     '/funding/programmes/digital-funding': '/funding/programmes/digital-fund',
-    '/funding/scotland-portfolio': '/funding/programmes?location=scotland',
+    '/funding/scotland-portfolio/three-approaches': '/funding/funding-guidance/three-approaches-scotland',
     '/funding/search-past-grants-alpha': '/funding/grants',
-    '/guidancetrackingprogress': '/funding/funding-guidance/applying-for-funding/tracking-project-progress/guidance-on-tracking-progress',
```

Comment:
`guidancetrackingprogress` is a vanity url and is already in the CMS so can be removed